docs/tutorial/population_dynamics.rst

```diff
@@ -157,6 +174,60 @@ the distribution values are directly available through the properties `Dwelltime
+a DNA strand:
+
+.. math::
+
+    \left[ \mathrm{DNA} \cdot \mathrm{protein} \right] \rightarrow \mathrm{DNA} + \mathrm{protein}
+
+This reaction is characterized by a rate constant :math:`k_\mathrm{off}` known as the dissociation rate constant with units of
+:math:`\mathrm{sec}^{-1}`.
+
+Second order reactions which are dependent on *two* reactants can also be determined in this way if certain conditions are met. Specifically,
+if the concentration of one reactant is much greater than that of the other, we can apply the *first order approximation*. This approximation
+assumes the concentration of the more abundant reactant remains approximately constant throughout the experiment and therefore does
+not contribute to the reaction rate. This condition is often met in single-molecule experiments; for example in a typical C-Trap experiment,
+the concentration of a protein in solution on the order of nM is significantly higher than the concentration of the single trapped tether.
+
+A common example of this is the binding of a protein to a DNA strand:
+
+.. math::
+
+    \mathrm{DNA} + \mathrm{protein} \rightarrow \left[ \mathrm{DNA} \cdot \mathrm{protein} \right]
+
+This reaction is described by the second order association rate constant :math:`k_\mathrm{on}` with units of :math:`\mathrm{M}^{-1}\mathrm{sec}^{-1}`.
+Under the first order approximation, this can be determined by fitting the appropriate dwell times to the exponential model and multiplying
+the resulting rate constant by the concentration of the protein in solution.
+
+.. note::
+
+    This procedure relies on having an accurate measurement of the bulk concentration, which can be difficult to determine when working
+    in the nanomolar regime, as nonspecific adsorption can lower the
+    effective concentration at the experiment.
+
+A warning on reliability and interpretation of multi-exponential kinetics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Care must also be taken when interpreting a mixture of exponential distributions. Sometimes, a double exponential distribution is required to accurately
+describe the observed data. This occurs when a system consists of two states with different kinetics that emit the same observable signal. For instance,
+the dissociation rate of a bound protein might depend on the microscopic conformation of the molecule that does not affect the emission intensity of
+an attached fluorophore used for tracking.
+
+However, in the setting of a limited number of observations, the optimization of the exponential mixture can
+become non-identifiable, meaning that there are multiple sets of parameters that result in near equal likelihoods. A good first check on the quality of
+the optimization is to run a bootstrap simulation (as described above) and check the shape of the resulting distributions. Very wide, flat, or skewed
+distributions can indicate that the model was not fitted to a sufficient amount of data.
```

Comment:
Maybe end the section using an advice on how to handle a bit fit, eg  'for processes that are best described by two exponentials, it may be necessary to acquire more data for a good fit'.